packages/react/src/themes/teams-dark/components/Menu/menuVariables.ts

```diff
@@ -1,9 +1,16 @@
-export interface MenuVariables {
-  color: string
-}
-
-export default (siteVars: any): MenuVariables => {
-  return {
-    color: siteVars.colors.white,
```

Comment:
Can you suggest why the color from here was removed?

Reply:
I removed the simplistic interface & imported the real reusable one. It still has the color attribute and all the other related ones too.